Step7/README.md

```diff
@@ -0,0 +1,80 @@
+# Step7
+Goal:
+* Download the Custom Vision model
+* Use it locally with tensorflow.js
+
+## Download the model locally
+
+- Press Export
+
+![Press Export](../assets/screenshots/0_export_model_1.png "Press Export")
+
+- Select Tensorflow
+
+![Select Tensorflow](../assets/screenshots/0_export_model_2.png "Select Tensorflow")
+
+- Select Tensorflow.js and press Download
+
+![Select Tensorflow.js and press Download](../assets/screenshots/0_export_model_3.png "Select Tensorflow.js and press Download")
+
+- Extract all the files in `Step7/tf-js-model`
+
+- In the folder `Step7` run
+
+```
+npm install @microsoft/customvision-tfjs-node
+```
+
+## Changes in public/js/app.js
+```javascript
+/* Add another parameter to "toBlob" function to enable compatibility with Tensorflow.js */
+canvasElement.toBlob(function(blob) {
+    request.send(blob);
+}, "image/png");
+```
+
+## Changes in index.js
+```javascript
+const fs = require('fs');
+const express = require('express');
+const app = express();
+const PORT = process.env.PORT || 1337;
+const bodyParser = require('body-parser')
+const cvstfjs = require("@microsoft/customvision-tfjs-node");
+
+...
+
+app.post('/predict', async (request, response) => {
+    const imageData = request.body;
+
+    let labels;
+    try {
+        const data = (await fs.readFileSync('tf-js-model/labels.txt')).toString();

```

Comment:
Let's init the model once on line 46, so we don't have to wait for the model to load weights each time we want to predict something